vspc/server.py

```diff
@@ -195,6 +198,24 @@ async def save_to_log(self, uuid, data):
         fpath = os.path.join(CONF.serial_log_dir, uuid)
         async with aiofiles.open(fpath, 'ab') as f:
             await f.write(data)
+        if CONF.log_file_size and os.path.getsize(fpath) > CONF.log_file_size:
```

Comment:
Who awaits the `getsize()`?

Reply:
everyone, it's a cheap operation, but better to use `ospath` from `aiofiles`

Reply:
but I won't use it because it's not implemented in the library yet.